lib/web3/formatters.js

```diff
@@ -159,6 +159,9 @@ var inputPostFormatter = function(post) {
     post.priority = utils.fromDecimal(post.priority);
 
     // fallback
+    if (post.topics == undefined) {
+        post.topics = []
+    }
     if (!utils.isArray(post.topics)) {
```

Comment:
Can you remove this? As the line below already takes care of creating and array if non is given.